terraria-tmodloader/.skip

```diff
@@ -1 +0,0 @@
-
```

Comment:
Please leave this file as it is used for our testing suite, unless tmodloader is able to run as-is with no changes at all and doesn't need a license/password. I think last time we ran this, it had issues related to that.